### ISSUE TITLE
AndroidManifest.xml specifies application label, which can cause manifest merge problems

### ISSUE THREAD

**Issue by al-broco** [NordicSemiconductor/nRF-Logger-API#5]
The library's AndroidManifest.xml file specifies the application@label attribute, which I do not think it should. This attribute is the responsibility of the app, not a library used by the app.

How to reproduce: Create an app, specify the application@label property in its AndroidManifest.xml like so:

```
<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="...">
 ...
 <application
     ...
      android:label="@string/general.app_name"
```

Include the log lib as a dependency like so:

```
compile 'no.nordicsemi.android:log:2.1.1'
```

This will result in a build error like:

```
Error:
	Attribute application@label value=(@string/general.app_name) from AndroidManifest.xml:25:9-49
	is also present at [no.nordicsemi.android:log:2.1.1] AndroidManifest.xml:11:18-50 value=(@string/app_name).
	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml:5:5-27:19 to override.
```

**Comment by philips77** [comment]
Fixed in version 2.1.2.

**Comment by EVERCLEAR11** [comment]
Please fix?
